subprojects/core/src/test/groovy/org/gradle/api/internal/file/archive/TarCopyActionSpec.groovy

```diff
@@ -111,7 +111,7 @@ public class TarCopyActionSpec extends Specification {
     }
 
     private TestFile initializeTarFile(final TestFile tarFile, final ArchiveOutputStreamFactory compressor) {
-        action = new TarCopyAction(tarFile, compressor);
+        action = new TarCopyAction(tarFile, compressor, false);
```

Comment:
We do, and we do it [here](https://github.com/gradle/gradle/blob/a1e022d330e6de2db28ff65d4c63df25f4a6a588/subprojects/core/src/main/java/org/gradle/api/tasks/bundling/Tar.java#L46-L46). Just the test is currently passing false.